### PR TITLE
Remove automatically scrolling into view functionality

### LINKED ISSUE #1545: [Bug]: Dashboard keeps scrolling to bottom of project when Models Tab is expanded.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Deploy type

OpenDataHub core version (eg. `v1.6.0`)

### Version

1.6.0

### Current Behavior

https://github.com/opendatahub-io/odh-dashboard/assets/16117276/75d32d2f-795d-4b50-9fc1-e7169459acda

Watch the video to see the issue.

At first, I can start/stop the workbench and nothing moves. 

If I expand the model server to see the 3 models, any activity I try to do, the page auto-scrolls to the bottom.

### Expected Behavior

There is no autoscroll to the bottom of the page

### Steps To Reproduce

1. Create enough clutter in the DS project to have a long page
2. extend the model server to see the multiple models being served. 
3. try to scroll to the top of the page, and start/stop a workbench

### Workaround (if any)

_No response_

### What browsers are you seeing the problem on?

_No response_

### Anything else

_No response_

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**
Closes #1545 

## Description

The div will automatically scroll into view because we added that feature somehow. But I think it's not needed because users can always scroll the screen by themselves, auto-scroll will only destroy some of the layouts and behaviors and cause unexpected results.

@kywalker-rh Could you confirm this?
The context: When you click on `Deployed models` on project details page, it will expand the models table, and the page will scroll to show the whole table. However, this auto scroll could cause problems. So I removed it, no when clicking on `Deployed models`, the page will not auto scroll anymore. Is that acceptable?

## How Has This Been Tested?

Follow the same instructions in the issues, you will find the issue disappeared.

## Test Impact

N/A, layout issue.

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [x] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by kywalker-rh** [comment]
@vconzola - Can you take a look at this? I think you were the one that worked on the design originally.

**Comment by DaoDaoNoCode** [comment]
@vconzola I can make the auto expand happen when you newly deploy a model.
What this PR is trying to fix is actually another issue, which is when you expand the deployed model section, it will scroll to the bottom of the page because we want it to show the whole model table. However, this is problematic because the auto scroll to the bottom can cause layout issues. What I am asking is, do we still need the auto-scroll? Can we remove it?
This is the expected behavior after the fix: When users deploy the model, the `Deployed models` section will automatically expand, but it will not cause any auto-scroll of the page, you may only be able to see half of the table and you can scroll down to see the whole table. Do you think it's acceptable?

**Comment by vconzola** [comment]
@DaoDaoNoCode The problem I was seeing before is that even with auto-expand when a new model was deployed the expansion area was displayed "below the fold" so the user didn't see it without scrolling and therefore didn't know the model was deployed. Can we do any sort of auto-scrolling to ensure the expansion is visible without the user needing to manually scroll? Or is auto-scrolling in general not an option?

**Comment by DaoDaoNoCode** [comment]
@vconzola Let me give it a try.

**Review by lucferbux — APPROVED** [review]
/lgtm

mmmm so the issue was that every re-render we were scrolling again to the element right? how curious 🤔

**Comment by DaoDaoNoCode** [comment]
@lucferbux Haha I guess it was me who wrote this erroneous functionality... But I cannot recall what I was thinking at that time 🤷‍♂️

**Review by Gkrumbach07 — APPROVED** [review]

File: frontend/src/pages/modelServing/screens/projects/ServingRuntimeTableExpandedSection.tsx [hunk 1]
```diff
@@ -59,7 +59,9 @@ const ServingRuntimeTableExpandedSection: React.FC<ServingRuntimeTableExpandedSe
           ) : (
             <EmptyInferenceServicesCell onDeployModel={onDeployModel} />
           )}
-          <ScrollViewOnMount />
+          <ScrollViewOnMount
+            shouldScroll={activeColumn === ServingRuntimeTableTabs.DEPLOYED_MODELS}
+          />
```
[the remark this answers is not in the document]
Comment (DaoDaoNoCode) [reply frontend/src/pages/modelServing/screens/projects/ServingRuntimeTableExpandedSection.tsx:64]: Hmm, I totally forgot I could unmount it...

**Pull request merged**